src/Raven.Studio/typescript/common/shell/databasesManager.ts

```diff
@@ -44,7 +44,8 @@ class databasesManager {
     init(): JQueryPromise<Raven.Client.ServerWide.Operations.DatabasesInfo> {
         return this.refreshDatabases()
             .done(() => {
-                this.activateBasedOnCurrentUrl();
+                const dbNameFromUrl = appUrl.getDatabaseNameFromUrl();
+                this.activateBasedOnCurrentUrl(dbNameFromUrl, true);
```

Comment:
why we always pass true here?

Reply:
Removed this part for now - seems to work w/o this